.github/workflows/microsite.yml

```diff
@@ -7,13 +7,28 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-18.04
+    runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Build docs
+        run: |
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+          brew install sourcekitten
+          sudo gem install jazzy bundler
+          chmod +x generate-documentation.sh
```

Comment:
Is this chmod needed or can we check-in the permission change into git (if not already)?

Reply:
It seems the flag is already set, I will remove this line!